### ISSUE TITLE
Action Executor ignores field value for the preview function

### ISSUE THREAD

**Issue by romaninsh** [atk4/ui#969]
When action defines fields=['name'] - executor asks for that name and sets it before executing action (as expected)

If you have a preview, however, the value is not set for the preview:

<img width="767" alt="Screenshot 2020-02-26 at 18 44 24" src="https://user-images.githubusercontent.com/453929/75376607-3c80d100-58c8-11ea-9f9d-7f8f41d59b1d.png">

<img width="1184" alt="Screenshot 2020-02-26 at 18 44 28" src="https://user-images.githubusercontent.com/453929/75376615-3f7bc180-58c8-11ea-9b5f-4a56943466e9.png">

<img width="974" alt="Screenshot 2020-02-26 at 18 44 37" src="https://user-images.githubusercontent.com/453929/75376622-4276b200-58c8-11ea-9902-5e5f8fdbb739.png">

<img width="956" alt="Screenshot 2020-02-26 at 18 44 41" src="https://user-images.githubusercontent.com/453929/75376636-4571a280-58c8-11ea-879f-4fb268df9b22.png">

<img width="316" alt="Screenshot 2020-02-26 at 18 44 45" src="https://user-images.githubusercontent.com/453929/75376642-47d3fc80-58c8-11ea-8904-352c3bf16463.png">

Code:

```

class Client extends Model
{
    public $table = 'client';

    public function init()
    {
        parent::init();

        $this->addField('name');

        $this->addAction('send_statement', [ 'fields'=>['name'], 'preview'=>'send_statement_preview']);
    }

    public function send_statement_preview() {
        return 'here goes your statement '.$this['name'];

    }
    public function send_statement() {
        /*
        if (!isset($this->app->mailer)) {
            throw new ValidationException('ouch');
        }
        */
        return 'sent to '.$this['name'];
    }
}
```

**Comment by ibelar** [comment]
@romaninsh - You should be using argument for this.

```
class Client extends Model
{
    public $table = 'client';

    public function init()
    {
        parent::init();

        $this->addField('name');

        $this->addAction('send_statement', [ 'args'=>['name' => ['required'=> true]]], 'preview'=>'send_statement_preview']);
    }

    public function send_statement_preview() {
        return 'here goes your statement '.$this['name'];

    }
    public function send_statement($m, $name) {
        /*
        if (!isset($this->app->mailer)) {
            throw new ValidationException('ouch');
        }
        */
        $m->tryLoadBy('name', $name);
        return 'sent to '.$this['name'];
    }
}
```

Fields will only set form for direct model editing/saving.

**Comment by ibelar** [comment]
@romaninsh - nevermind previous comment. Fix in #970
